playbooks/common/openshift-cluster/additional_config.yml

```diff
@@ -32,6 +32,11 @@
   - role: os_env_extras
     when: openshift.common.deployment_type == 'online'
 
+- name: Gather hosted facts
+  hosts: oo_first_master
+  roles:
+  - role: openshift_hosted_facts
```

Comment:
This play sets up hosted facts for the following play. An alternative would be to add hosted facts as a dep to the `openshift_master` role.